packages/react-vapor/src/components/datePicker/tests/DatePickerReducers.spec.ts

```diff
@@ -894,6 +894,22 @@ describe('Date picker', () => {
 
                 expect(newDatePicker.appliedUpperLimit).toBe(oldDatePicker.lowerLimit);
             });
+
+            it('should set the upperLimit to the end of the day value if the state is set as a range and the upperLimit is undefined', () => {

```

Comment:
I did not followed all the discussion for this change but are we sure it will not cause issues that the upper limit is now set to the end of the day and not now? :thinking:

Reply:
It causes an issue actually that the date was set a now, because you add a validation if the upperLimit and the lowerLimit were equal. The user could set themself as now still though. Do you think there an instance that it could causes an issue (I might have oversight something).

Reply:
I don't know that's why I ask :see_no_evil: 

But Like you already said, if the props is not used it wont be an issue :shrug: